### PR TITLE
Hubert/ Allowing users to select previously disabled tax residencies

### PR DESCRIPTION
**Pull request by hubert-deriv**

### CODE REVIEW

File: src/javascript/app/common/account_opening.js [hunk 1]
```diff
@@ -155,7 +155,7 @@ const AccountOpening = (() => {
                 }
 
                 const $tax_residence_select = $('#tax_residence');
-                $tax_residence_select.html($options_with_disabled.html());
+                $tax_residence_select.html($options.html());
```
Comment (negarn) [inline src/javascript/app/common/account_opening.js:158]: they keep asking us to change this every few weeks 😂

File: src/javascript/app/pages/user/account/settings/personal_details.js [hunk 2]
```diff
@@ -157,7 +165,7 @@ const PersonalDetails = (() => {
             displayChangeableFields(data);
             $(real_acc_elements).setVisibility(1);
             showHideTaxMessage();
-            CommonFunctions.getElementById('tax_information_form').setVisibility(shouldShowTax(get_settings));
+            CommonFunctions.getElementById('tax_information_form').setVisibility(1);
```
Comment (negarn) [inline src/javascript/app/pages/user/account/settings/personal_details.js:168]: if we want to make it visible for all real accounts, should we handle it as part of the `real_acc_elements` then?

File: src/javascript/app/pages/user/metatrader/metatrader.ui.js [hunk 3]
```diff
@@ -644,6 +644,7 @@ const MetaTraderUI = (() => {
     const handleNewAccountUI = (action, acc_type, $target) => {
         current_action_ui = action;
 
+        // const get_financial_container = $('#rbtn_financial_financial.selected').length === 0 ? true : false;
```
Comment (negarn) [inline src/javascript/app/pages/user/metatrader/metatrader.ui.js:647]: ?

File: src/javascript/app/common/account_opening.js (hunk 1 above)
Comment (hubert-deriv) [reply src/javascript/app/common/account_opening.js:158]: We'll see what happens I guess, haha.

File: src/javascript/app/pages/user/metatrader/metatrader.ui.js (hunk 3 above)
Comment (hubert-deriv) [reply src/javascript/app/pages/user/metatrader/metatrader.ui.js:647]: Removed this.

File: src/javascript/app/pages/user/account/settings/personal_details.js (hunk 2 above)
Comment (hubert-deriv) [reply src/javascript/app/pages/user/account/settings/personal_details.js:168]: Done, I added the RealAcc class to this element in personal_details and removed this line of code.

**Review by negarn — APPROVED** [review]

**Review by negarn — DISMISSED** [review]

**Pull request merged**